identity/resource_group.go

```diff
@@ -124,49 +101,38 @@ func ResourceGroup() *schema.Resource {
 				ForceNew: true,
 				Required: true,
 			},
-			"allow_cluster_create": {
-				Type:     schema.TypeBool,
-				Optional: true,
-			},
-			"allow_sql_analytics_access": {
-				Type:     schema.TypeBool,
-				Optional: true,
-			},
-			"allow_instance_pool_create": {
-				Type:     schema.TypeBool,
-				Optional: true,
-			},
 			"url": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
 		},
 	}
-}
-
-func isGroupClusterCreateEntitled(group *ScimGroup) bool {
-	for _, entitlement := range group.Entitlements {
-		if entitlement.Value == AllowClusterCreateEntitlement {
-			return true
+	//Add entitlements to the schema
+	for _, entitlementField := range entitlementFields {
```

Comment:
1) do not introduce a variable for resource, where you can introduce a variable just for schema :)
2) make sure that workspace access has true by default. otherwise thousands of users won't have access to workspaces anymore.